main.go

```diff
@@ -0,0 +1,133 @@
+package main
```

Comment:
Nit: renaming the file was unnecessary, it is quite common to have the main file named something else than main in go.